guides/hosting/infrastructure/elasticsearch/elasticsearch-setup.md

```diff
@@ -9,58 +9,61 @@ nav:
+Elasticsearch installation and configuration greatly depend on your operating system and hosting provider. You will find extensive documentation online regarding the installation and configuration of Elasticsearch on most common Linux distributions. Some hosting providers might also provide specific documentation regarding this subject. Installation on macOS or Windows is also possible but not officially supported.
 
-The current Shopware 6 integration is designed to work with the out-of-the-box configuration of Elasticsearch. This does not mean, of course, that these are the best settings for a production environment. Although they will affect performance and security, the settings you choose to use on your Elasticsearch setup will be mostly transparent to your Shopware installation. The best setting constellation for your shop will greatly depend on your server setup, the number, and structure of products, and replication requirements, to name a few. In this document, we can't give you specific examples for your setup, but provide you with hints and basics you might need to choose your perfect setup. More detailed information can be found on the official [Elasticsearch](https://www.elastic.co/guide/index.html)  documentation page.
+The current Shopware 6 integration is designed to work with the out-of-the-box configuration of Elasticsearch. This does not mean, of course, that these are the best settings for a production environment. Although they will affect performance and security, the settings you choose to use on your Elasticsearch setup will be mostly transparent to your Shopware installation. The best setting constellation for your shop will greatly depend on your server setup, the number, and structure of products, and replication requirements, to name a few. In this document, we can't give you specific examples for your setup, but provide you with hints and basics you might need to choose your perfect setup. More detailed information can be found on the official [Elasticsearch](https://www.elastic.co/guide/index.html) documentation page.
 
 ### Elasticsearch server setup
 
 Elasticsearch is meant to be used as a cluster setup so it can scale properly and provide you with reliability.
 In this cluster, you can choose how many nodes you want to use and which different type each node in the cluster shall have.
-A one node cluster is only meant for development or test environments because it can't scale at all and does not give you any more reliability.
+A one-node cluster is only meant for development or test environments because it can't scale at all and does not give you any more reliability.
```

Comment:
```suggestion
A one-node cluster should only be used for development or test environments, because it can't scale and does not provide additional reliability.
```